mayastor/src/bin/mayastor-client/bdev_cli.rs

```diff
@@ -37,12 +37,10 @@ pub fn subcommands<'a, 'b>() -> App<'a, 'b> {
     let share = SubCommand::with_name("share")
         .about("share the given bdev")
         .arg(Arg::with_name("name").required(true).index(1))
-        .arg(
-            Arg::with_name("protocol")
-                .help("the protocol to used to share the given bdev")
-                .required(false)
-                .default_value("nvmf")
-                .value_names(&["nvmf", "iscsi"]),
+        .arg(Arg::from_usage("<protocol> 'the protocol to used to share the given bdev'")
```

Comment:
Could you please keep the `Arg::with_name` style here for consistency? You can see above we do not use `from_usage` there.

Reply:
modules the comment from @Hoverbear; LGTM.

Reply:
Oh, just saw the follow up commit =)